### DISCUSSION TITLE
allow commas to count as decimal periods

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18271] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/78441708/168454386-157e2219-a28a-48d0-a964-eed8e64f645a.png)

this should correlate to 1.4; currently it correlates to 14 which goes down to 10.

**Comment by smoogipoo** [comment, marked as answer]
`1,4` is not a valid number in some (esp. English) locales. Instead of this, the textbox should respect the user's current culture.

**Reply by peppy** [reply]
Probably best to make a tracking issue over on framework.

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu-framework/issues/5176
